lnrpc/rpc.proto

```diff
@@ -268,10 +268,14 @@ message ActiveChannel {
     int64 total_satoshis_received = 10 [ json_name = "total_satoshis_received" ];
     uint64 num_updates = 11 [ json_name = "num_updates" ];
 
-    repeated HTLC pending_htlcs = 12 [ json_name = "pending_htlcs" ];
+    ChannelStatus status = 12 [ json_name = "status" ];
```

Comment:
Hmm, I don't think this field belongs here. The status of an active channel is "open". The `PendingChannelResponse` proto should've been tweaked rather than this proto.